test/common/priority_validate.go

```diff
@@ -149,6 +149,10 @@ func TestPriorityClass(t TestingTB, ctx *TestingContext) {
 		}
 	}
 	for _, d := range priorityDeployments() {
+		// skip  the threescale-operator-controller-manager deployment
+		if strings.Contains(d.Name, "threescale-operator-controller-manager") {
```

Comment:
why is this done?

Reply:
@MStokluska , it was done because of problems in E2E. Change was done a month ago, so I will recheck it, and will process accordingly, and provide more info. Thank you

Reply:
thanks

Reply:
"skip  the threescale-operator-controller-manager deployment" -   removed, as retested and E2E passed with  original code. So just rerunning again after cleanup. Thank you for comment